redash/query_runner/jql.py

```diff
@@ -45,19 +63,27 @@ def parse_issue(issue):
 
             if 'watchCount' in v:
                 result[k] = v['watchCount']
-        # elif isinstance(v, list):
-        #     pass
-        else:
-            result[k] = v
+    
+    elif isinstance(v, list):
+        listValues = []
+        for listItem in v:
+            if isinstance(listItem, dict):
+                if 'member' in mapping:
+                    listValues.append(listItem[mapping['member']])
```

Comment:
What if `mapping['member']` not in `listItem`? Better to use `listItem.get` or check for this case and raise an explicit error.